docs/ca/dev/preparation-for-going-live.md

```diff
@@ -23,191 +23,108 @@ We've divided the preparation into approximate timeframes, and you can adjust th
+| APP-27 | Application | Check environment variables                            | Double-check whether you have all needed environment variables and parameter store values set up. Remember that this has some lead time on our side. If you are still missing parameters, create them.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
What's the lead time for?

Reply:
Good point. 
Hm, I would say that it may require additional time. 
But it should not, so maybe we can even remove the "Remember that this has some lead time on our side." sentence